src/plot_topoplotseries.jl

```diff
@@ -32,13 +31,16 @@ Multiple miniature topoplots in regular distances.
 - `positions::Vector{Point{2, Float32}} = nothing`\\
     Specify channel positions. Requires the list of x and y positions for all unique electrode.
 - `interactive_scatter = nothing`\\
-    Enable interactive mode. \\ 
+    Enable interactive mode.\\ 
     If you create `obs_tuple = Observable((0, 0, 0))` and pass it into `interactive_scatter` you can change observable indecies by clicking topopplot markers.\\
     `(0, 0, 0)` corresponds to the indecies of row of topoplot layout, column of topoplot layout and channell. 
+- `topoplot_axes = (;)`\\
+    Here you can flexibly change configurations of topoplots.\\
+    To see all options just type `?Axis` in REPL.
 - `mapping.x = :time`\\
-    Specification of x value, could be any contionous variable. 
+    Specify x-value. Can be any continuous variable.
```

Comment:
this is a bit missleading, you should rather describe the `mapping = (;)` relationship, right?

Reply:
or you could make a section "## facet options"